docs/client/advanced/features/plan_only.md

```diff
@@ -0,0 +1,101 @@
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+# Plan Only Execution Mode
+
+plan only execution mode is implemented through Spark extensions, so it is only supported Spark engine so far.
+
+It can print Spark's parser, analyze or optimize phases execution plan, and supports session-level and operation-level parameter settings, which are used in the following ways.
```

Comment:
the `session-level` and `operation-level` sound a little confused. In general, we call them: `application-level` and `session-level`